python/redmonster/physics/zpicker2.py

```diff
@@ -138,11 +139,17 @@ def classify_obj(self, zfindobjs, zfitobjs, flags, rchi2threshold=0.005):
                             fiberminvecs.append(
                                     zfitobjs[itemp].minvectors[ifiber][imin])
                         except IndexError as e:
-                            print "%r" % e
-                            fibermins.append( \
-                                    n.max(zfitobjs[itemp].chi2vals[ifiber]) / \
-                                    (self.dof[ifiber] - zfindobjs[itemp].npoly))
+                            #print "%r" % e
+                            #fibermins.append( \
+                            #        n.max(zfitobjs[itemp].chi2vals[ifiber]) / \
+                            #        (self.dof[ifiber] - zfindobjs[itemp].npoly))
+
+                            # this is ok, might be no solution
+                            #print "WARNING no z fitted for fiber #%d, class #%d, zid #%d"%(ifiber,itemp,imin)
+
+                            fibermins.append(100000.)
```

Comment:
I updated this to check for 0-length list before taking the max, falling back to only using a large value if needed.